internal/core/application/service.go

```diff
@@ -545,13 +569,48 @@ func (s *Service) WhenNextSettlement(ctx context.Context) time.Time {
 	return s.schedulerSvc.WhenNextSettlement()
 }
 
-func (s *Service) ConnectLN(ctx context.Context, connectUrl string) error {
-	if strings.HasPrefix(connectUrl, "clnconnect:") {
-		s.lnSvc = cln.NewService()
+func (s *Service) ConnectLN(ctx context.Context, lnUrl string) error {
+	if len(lnUrl) == 0 {
+		settings, err := s.dbSvc.Settings().GetSettings(ctx)
+		if err != nil {
+			log.WithError(err).Warn("failed to get settings")
+			return err
+		}
+
+		if settings.LnConnectionOpts == nil {
+			return fmt.Errorf("no LN connection options found, please provide a valid LN URL")
```

Comment:
let's put in evidence we expect a connect url in this case? (i mean `lndconnect://` or `clnconnect://` urls)

```suggestion
			return fmt.Errorf("no LN connection options found, please provide a valid LN connect URL")
```